ddtrace/appsec/_iast/__init__.py

```diff
@@ -92,6 +92,12 @@ def enable_iast_propagation():
     global _iast_propagation_enabled
     if _iast_propagation_enabled:
         return
+    # We need to preload the package_distributions because if we call importlib_metadata inside a module hook
+    # it raises a circular-import
+    from ddtrace.internal.packages import get_package_distributions
+
+    get_package_distributions()
```

Comment:
Question, where is this actually used?

Reply:
It's cached because it's expensive and should always return the same value, so it can be called early at any point.
However, it seems to be used only in internal.debug ??